CSS/ex_9/index.html

```diff
@@ -80,21 +77,22 @@
         <img src="http://placehold.it/50x50" />
       </div>
     </nav>
+
     <main>
-        <img src="google-logo.png" alt="Google's Logo" class="google-logo">
-        <form class="search-container">
-            <div class="search-container">
-                <img src="search-icon.png" alt="Search Icon"/>
-                <input type="text" />
-                <img src="voice-icon.png" alt="Voice Icon"/>
-            </div>
+      <img src="google-logo.png" alt="" class="google-logo" />
 
-            <div class="button-container">
-                <button>Google Search</button>
-                <button>I'm Feeling Lucky</button>
-            </div>
-        </form>
+      <form>
+        <div class="search-container">
+          <img src="search-icon.png" alt="Search Icon" />
+          <input type="text" />
+          <img src="voice-icon.png" alt="Voice Icon" />
+        </div>
 
+      <div class="button-container">
+         <button>Google Search</button>
+       <button>I'm Feeling Lucky</button>
+     </div>
+      </form>
```

Comment:
You could also try doing exercises here. It's a good practice.
https://github.com/chriztie/react-comit/tree/master/Prerequisites/CSS/09-css-google#exercise-instructions